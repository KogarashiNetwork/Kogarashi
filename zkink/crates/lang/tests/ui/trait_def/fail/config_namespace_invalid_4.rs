use ink_lang as ink;

#[ink::trait_definition(namespace = true)]
pub trait TraitDefinition {
    #[ink(message)]
    fn message(&self);
}

fn main() {}
