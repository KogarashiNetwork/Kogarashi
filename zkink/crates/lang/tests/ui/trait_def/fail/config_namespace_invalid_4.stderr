error: expected a string literal for `namespace` ink! trait definition configuration argument
 --> tests/ui/trait_def/fail/config_namespace_invalid_4.rs:3:25
  |
3 | #[ink::trait_definition(namespace = true)]
  |                         ^^^^^^^^^^^^^^^^
